extended/src/test/java/apoc/ml/OpenAIOpenLMIT.java

```diff
@@ -31,12 +34,169 @@ public class OpenAIOpenLMIT {
+                assertEquals(1, wordCount.length);
+            });
+}
+
+@Test
+public void completionWithAnthropicCustomVersion() {
+    String anthropicApiKey = System.getenv("ANTHROPIC_API_KEY");
+    Assume.assumeNotNull("No ANTHROPIC_API_KEY environment configured", anthropicApiKey);
+
+    Map<String, Object> conf = Map.of(
+            ENDPOINT_CONF_KEY, "https://api.anthropic.com/v1/messages",
+            API_TYPE_CONF_KEY, OpenAIRequestHandler.Type.ANTHROPIC.name(),
+            MAX_TOKENS, 1024,
+            ANTHROPIC_VERSION, "2023-06-01"
+    );
+    testCall(db, COMPLETION_QUERY_EXTENDED,
+            Map.of("conf", conf, "apiKey", anthropicApiKey),
+            (row) -> {
+                var result = (Map<String,Object>) row.get("value");
+                var contentList = (List<Map<String, Object>>) result.get("content");
+                Map<String, Object> content = contentList.get(0);
+                String generatedText = (String) content.get("text");
+                assertTrue(generatedText.toLowerCase().contains("blue"),
+                        "Actual generatedText is " + generatedText);
+            });
+}
+
+@Test
+public void completionWithAnthropicWrongVersion() {
+    String anthropicApiKey = System.getenv("ANTHROPIC_API_KEY");
+    Assume.assumeNotNull("No ANTHROPIC_API_KEY environment configured", anthropicApiKey);
+
+    Map<String, Object> conf = Map.of(
+            ENDPOINT_CONF_KEY, "https://api.anthropic.com/v1/messages",
+            API_TYPE_CONF_KEY, OpenAIRequestHandler.Type.ANTHROPIC.name(),
+            MAX_TOKENS, 1024,
+            ANTHROPIC_VERSION, "ajeje"
+    );
+
+    assertFails(
+            db,
+            COMPLETION_QUERY_EXTENDED,
+            Map.of("conf", conf, "apiKey", anthropicApiKey),
+            "Server returned HTTP response code: 400 for URL: https://api.anthropic.com/v1/messages"
+    );
+}
+
+@Ignore
+@Test
+public void completionWithAnthropicStream() {
+    String anthropicApiKey = System.getenv("ANTHROPIC_API_KEY");
+    Assume.assumeNotNull("No ANTHROPIC_API_KEY environment configured", anthropicApiKey);
+
+    Map<String, Object> conf = Map.of(
+            ENDPOINT_CONF_KEY, "https://api.anthropic.com/v1/messages",
+            API_TYPE_CONF_KEY, OpenAIRequestHandler.Type.ANTHROPIC.name(),
+            MAX_TOKENS, 256,
+            "stream", true
+    );
+    testCall(db, COMPLETION_QUERY_EXTENDED,
+            Map.of("conf", conf, "apiKey", anthropicApiKey),
+            (row) -> {
+                var result = (Map<String,Object>) row.get("value");
+                var contentList = (List<Map<String, Object>>) result.get("content");
+                Map<String, Object> content = contentList.get(0);
+                String generatedText = (String) content.get("text");
+                assertTrue(generatedText.toLowerCase().contains("blue"),
+                        "Actual generatedText is " + generatedText);
+            });
+}
+
```

Comment:
Mmm in effetti andrebbe anche aggiunto un test utilizzando `CHAT_COMPLETION_QUERY`, vedi come sta fatto il test chatCompletion() ad esempio in OpenAIIT.

E quindi andrebbe anche aggiunto sulla documentazione, quindi tipo: 
```
CALL apoc.ml.openai.chat([
  {role:"system", content:"Only answer with a single word"},
  {role:"user", content:"What planet do humans live on?"}
], 
$anthropicApiKey,
{apiType: 'ANTHROPIC', max_tokens: 1024})
```

Reply:
Ne basta uno solo, tanto gli altri casi sono già testati tramite la `COMPLETION_QUERY_EXTENDED`

Reply:
Oltre al test case con il tarallo.png